src/api/spec/controllers/source_project_meta_controller_spec.rb

```diff
@@ -15,4 +17,25 @@
     it { expect(response).to be_success }
     it { expect(Xmlhash.parse(response.body)['name']).to eq(project.name) }
   end
+
+  # rubocop: disable Layout/IndentHeredoc
+  describe 'PUT #update' do
+    let(:meta) do
+      <<~META
+      <project name="#{project.name}">
+        <title>My cool project</title>
+        <description></description>
+        <person userid="#{user.login}" role="maintainer" />
+      </project>
+      META
+    end
+    # rubocop: enable Layout/IndentHeredoc
```

Comment:
Why don't you just [indent](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/IndentHeredoc) the heredoc instead?

Reply:
done

Reply:
I was using the ```<<-``` and I didn't see you were using ```<<~``` in this case. fixed